e2e/utils.go

```diff
@@ -203,7 +203,6 @@ func getSnapshot(path string) v1alpha1.VolumeSnapshot {
 func createCephfsStorageClass(c kubernetes.Interface, f *framework.Framework) {
 	scPath := fmt.Sprintf("%s/%s", cephfsExamplePath, "storageclass.yaml")
 	sc := getStorageClass(scPath)
-	sc.Parameters["pool"] = "myfs-data0"
```

Comment:
as this is optional, please add a test case with and without pool parameter.

@poornimag is there a way we can identify in the backend (via toolbox pod)this image is having pool parameter enabled?

Reply:
added a test for with and without the pool